Makefile

```diff
@@ -8,12 +8,6 @@ check:
 
 all: build check
 
-lint:
-	go install honnef.co/go/tools/cmd/staticcheck
-	staticcheck ./...
-	go install golang.org/x/lint/golint
-	golint ./...
-
```

Comment:
Q: shouldn't you leave the target and just remove the `golint` lines?

Reply:
@pierre-fastly good point, removing the target was an oversight -- will add it back 👍